is_core/generic_views/table_views.py

```diff
@@ -220,6 +224,9 @@ def _get_export_display(self):
     def _get_export_types(self):
         return self.export_types or self.core.get_export_types(self.request)
 
+    def _get_list_per_page(self):
+        return self.list_per_page or self.core.get_list_per_page(self.request) or IS_CORE_LIST_PER_PAGE
+
```

Comment:
Na normálním view to funguje tak, že nejvyšší prioritu má view, pak ISCore a až pak konstanta v settings.